Clarify description of `safe` option (#8354)
jekyll/jekyll: docs/_docs/configuration/options.md

      <td>
        <p class="name"><strong>Safe</strong></p>
        <p class="description">

>           Disable <a href="/docs/plugins/">plugins</a>, caching to disk
>           and ignore symbolic links.  </br>
>           Whitelisted plugins are not disabled.

Reviewer: How about rephrasing to the following?
```suggestion
          Disable <a href="/docs/plugins/">non-whitelisted plugins</a>, caching to disk
          and ignore symbolic links.
```

Author: I like it. The only reason I had it separate was to link "whitelisted" to the config page for whitelisting.

Reviewer: I don't think we have a dedicated section on the `whitelist` configuration..
If that's indeed the case, I think you can add a *note* to the page at `/docs/plugins/` informing the visitor about the whitelist..
(*it can be part of this PR itself*)
